src/helpers/dnsutil.js

```diff
@@ -6,43 +6,50 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-import { DNSParserWrap as Dns } from "../dns-operation/dnsOperation.js"
-import * as envutil from "./envutil.js"
+import { DNSParserWrap as Dns } from "../dns-operation/dnsOperation.js";
+import * as envutil from "./envutil.js";
 
 // dns packet constants (in bytes)
 // A dns message over TCP stream has a header indicating length.
-export const dnsHeaderSize = 2
-export const dnsPacketHeaderSize = 12
-export const minDNSPacketSize = dnsPacketHeaderSize + 5
-export const maxDNSPacketSize = 4096
-export const cacheSize = 10000
+export const dnsHeaderSize = 2;
+export const dnsPacketHeaderSize = 12;
+export const minDNSPacketSize = dnsPacketHeaderSize + 5;
+export const maxDNSPacketSize = 4096;
 
-const minRequestTimeout = 5000 // 7s
-const defaultRequestTimeout = 15000 // 15s
-const maxRequestTimeout = 30000 // 30s
+export const dnsCloudflareSec = "1.1.1.2";
+export const dnsCacheSize = 10000;
+export const ttlGraceSec = 30;
```

Comment:
these `export`s were auto-added...?

Reply:
? Nope. Also, by convention, they are better named starting with `_` if not to be exported.

Reply:
I didn't certainly add these `export`s... Hm... Magic.

Noted: `_`.